schema/bom-1.6.schema.json

```diff
@@ -2320,16 +2318,36 @@
       ]
     },
     "version": {
-      "description": "A single version of a component or service.",
+      "description": "A single disjunctive version identifier, for a component or service.",
       "type": "string",
       "minLength": 1,
-      "maxLength": 1024
+      "maxLength": 1024,
+      "examples": [
+        "9.0.14",
+        "v1.33.7",
+        "7.0.0-M1",
+        "2.0pre1",
+        "1.0.0-beta1",
+        "0.8.15"
+      ]
     },
-    "range": {
+    "version-range": {
       "description": "A version range specified in Package URL Version Range syntax (vers) which is defined at https://github.com/package-url/purl-spec/VERSION-RANGE-SPEC.rst",
       "type": "string",
       "minLength": 1,
-      "maxLength": 1024
+      "maxLength": 1024,
+      "examples": [
+        "9.0.14",
+        "1.2.3|>=2.0.0|<5.0.0",
+        "0.0.0|0.0.1|0.0.2|0.0.3|1.0|2.0pre1",
+        ">=1.0.0-beta1|<=1.7.5|>=7.0.0-M1|<=7.0.7|>=7.1.0|<=7.1.2|>=8.0.0-M1|<=8.0.1",
+        ">=2.2.0|!= 2.2.1|<2.3.0"
+      ]
+    },
+    "range": {
```

Comment:
the name was too broad. therefore, it needed to be renamed to `version-range`

BUT `range` was en exisitn definition, which may have been used/referenced in JSON schemas downstream, by organizations that build supersets on out schema. 
Therefore, we cannot simply remove/rename it. 

SO it stayed as is, but is marked as deprecated and "inherits" from the new definition.